### PR TITLE
Fix flake8 warning (#741)

### PR DESCRIPTION
**Pull request by johnklee**
Fix flake8 warning (part2)

---
This change is [Reviewable](https://reviewable.io/reviews/google/mobly/745)

### CODE REVIEW

File: mobly/controllers/android_device.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def destroy(ads):
   for ad in ads:
     try:
       ad.services.stop_all()
-    except:
+    except Exception:
```
Comment (johnklee) [inline mobly/controllers/android_device.py:126]: Fix according to Flake8 rule [E722](https://www.flake8rules.com/rules/E722.html)
```
Do not use bare except, specify exception instead
```

File: mobly/controllers/android_device_lib/snippet_client.py [hunk 2]
```diff
@@ -313,7 +313,7 @@ def _read_protocol_line(self):
       # considering any blank output line to be EOF.
       line = line.strip()
       if (line.startswith('INSTRUMENTATION_RESULT:') or
-          line.startswith('SNIPPET ')):
+         line.startswith('SNIPPET ')):
```
Comment (johnklee) [inline mobly/controllers/android_device_lib/snippet_client.py:316]: Fix according to Flake8 rule [E129](https://www.flake8rules.com/rules/E129.html)
> Visually indented line with same indent as next logical line

File: mobly/controllers/attenuator.py [hunk 3]
```diff
@@ -64,7 +63,7 @@ def create(configs):
     attenuation_device = module.AttenuatorDevice(
         path_count=len(config[KEY_PATHS]))
     attenuation_device.model = attenuator_model
-    instances = attenuation_device.open(config[KEY_ADDRESS], config[KEY_PORT])
```
Comment (johnklee) [inline mobly/controllers/attenuator.py:67]: Fix according to Flake8 rule [F841](https://www.flake8rules.com/rules/F841.html)
> F841 local variable 'instances' is assigned to but never used

File: mobly/controllers/attenuator_lib/telnet_scpi_client.py [hunk 4]
```diff
@@ -70,7 +70,7 @@ def cmd(self, cmd_str, wait_ret=True):
       return None
 
     match_idx, match_val, ret_text = self._tn.expect(
-        [_ascii_string("\S+" + self.rx_cmd_separator)], 1)
+        [_ascii_string(r'\S+' + self.rx_cmd_separator)], 1)
```
Comment (johnklee) [inline mobly/controllers/attenuator_lib/telnet_scpi_client.py:73]: Fix according to Flake8 rule [W605](https://www.flake8rules.com/rules/W605.html)
> W605 invalid escape sequence '\S'

File: mobly/controllers/sniffer_lib/local/tshark.py [hunk 5]
```diff
@@ -37,7 +37,9 @@ def __init__(self, config_path, logger, base_configs=None):
   def get_descriptor(self):
     """See base class documentation
         """
-    return "local-tshark-{}-ch{}".format(self._interface)
+    # TODO: "local-tshark-{}-ch{}" needs two arguments
+    #       and only one `self._interface` is given here?
+    return "local-tshark-{0}-ch{0}".format(self._interface)
```
Comment (johnklee) [inline mobly/controllers/sniffer_lib/local/tshark.py:42]: Need clarification to see what we expect from the second argument. Above is just a temporal fix.

**Review by xpconanfan — CHANGES_REQUESTED** [review]

**Comment by johnklee** [comment]
> I think we should leave the monsoon, attenuator, and sniffer code as is.
> We plan to remove these code from Mobly repo as they are pretty old and not really used.

Thanks for the notification. Done in below commit:
https://github.com/johnklee/mobly/commit/f856b556b351d715ae6d652445a36e0fcdb4c784

**Review by xpconanfan — CHANGES_REQUESTED** [review]
I still see files for monsoon, sniffer lib, and iperf, which should be removed from the PR.

**Comment by johnklee** [comment]
> I still see files for monsoon, sniffer lib, and iperf, which should be removed from the PR.

Sorry for the inconvenience. I restore those files in below commit:
https://github.com/johnklee/mobly/commit/f5aa6e1bb78dea7af17a4007c84e1bbf1b8859f6

**Review by xpconanfan — APPROVED** [review]

**Pull request merged**